src/nunit.analyzers.codefixes/UseAssertEnterMultipleScope/UseAssertEnterMultipleScopeCodeFix.cs

```diff
@@ -0,0 +1,159 @@
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using NUnit.Analyzers.Constants;
+
+namespace NUnit.Analyzers.UseAssertEnterMultipleScope
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp)]
+    [Shared]
+    public class UseAssertEnterMultipleScopeCodeFix : CodeFixProvider
+    {
+        internal const string UseAssertEnterMultipleScopeMethod = "Use Assert.EnterMultipleScope method";
+
+        internal const string SystemThreadingTasksNamespace = "System.Threading.Tasks";
+        internal const string TaskTypeName = "Task";
+
+        public override ImmutableArray<string> FixableDiagnosticIds
+            => ImmutableArray.Create(AnalyzerIdentifiers.UseAssertEnterMultipleScope);
+
+        public sealed override FixAllProvider GetFixAllProvider()
+        {
+            return WellKnownFixAllProviders.BatchFixer;
+        }
+
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+
+            if (root is null)
+            {
+                return;
+            }
+
+            context.CancellationToken.ThrowIfCancellationRequested();
+
+            var node = root.FindNode(context.Span);
+
+            var expressionStatementSyntax = FindNearestParentOfType<ExpressionStatementSyntax>(node);
```

Comment:
There is a standard method that does the same:
```suggestion
            var expressionStatementSyntax = node.FirstAncestorOrSelf<ExpressionStatementSyntax>();
(node);
```